modules/cluster-logging-updating-logging.adoc

```diff
@@ -5,27 +5,29 @@
 [id="cluster-logging-updating-logging_{context}"]
 = Updating cluster logging
 
-After updating the {product-title} cluster, you can update cluster logging from 4.3 to 4.4 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.5 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator. You must update the Elasticsearch Operator before updating the Cluster Logging Operator and must update both the Elasticsearch Operator and the Cluster Logging Operator at the same time. Kibana is unusable when the Elasticsearch Operator has been upgraded but the Cluster Logging Operator has not been upgraded.
 
-ifdef::openshift-enterprise,openshift-webscale[]
 [IMPORTANT]
 ====
-Changes introduced by the new log forward feature modified the support for *out_forward* starting with the {product-title} 4.3 release. You create a ConfigMap to configure *out_forward*. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed.
-
-If you use the *out_forward* plug-in, before updating, you can copy your current `secure-forward.conf` section from the Fluentd ConfigMap and use the copied data when you create the `secure-forward` ConfigMap.
+If your cluster logging version is prior to 4.4, you must upgrade cluster logging to 4.4 before updating to 4.5. 
 ====
-endif::[]
 
 .Prerequisites
 
-* Update the cluster from 4.3 to 4.4.
+* Update the {product-title} cluster from 4.4 to 4.5. 
 
 * Make sure the cluster logging status is healthy:
 +
 ** All Pods are `ready`.
 ** Elasticsearch cluster is healthy.
 
-* Optionally, copy your current `secure-forward.conf` section from the Fluentd ConfigMap for use if you want to create the `secure-forward` ConfigMap. See the note above.
+* Back up your Elasticsearch and Kibana indices.
```

Comment:
Make a statement about backing up data but this command does not provide any value in relation to what we are trying to tell them.  In general its a good idea to backup data before upgrade but this command is simply going to list the indices